Makefile

```diff
@@ -144,21 +157,22 @@ kind-create: kind-clean
 OLM_IMAGE := quay.io/operator-framework/olm:local
 deploy:
 	$(KIND) load docker-image $(OLM_IMAGE) --name $(KIND_CLUSTER_NAME); \
-	$(HELM) install olm deploy/chart \
+	$(HELM) upgrade --install olm deploy/chart \
```

Comment:
Could you elaborate on why this change was needed?

Reply:
it was just convenient during development - if I just wanted to deploy and skip the whole kind create step